control/control.MicroOS.xml

```diff
@@ -263,6 +263,7 @@ textdomain="control"
        </globals>
 	      <software>	
             <default_patterns>microos_base microos_base_packagekit microos_defaults microos_hardware microos_gnome_desktop container_runtime bootloader</default_patterns>
+            <install_recommended config:type="boolean">true</install_recommended>
```

Comment:
Proposed fix:

```suggestion
            <minimalistic_libzypp_config config:type="boolean">false</minimalistic_libzypp_config>
```

Note: Untested!

Reply:
If you give me an easy way to test an install with this change, I'll be happy to report (and update my PR).

Reply:
found this code to create a DUD from my previous control XML experiments https://gist.github.com/lnussel/87f22be05ae8ff2def6d7b9df01f60ce

Reply:
I've tested this as dud but recommends aren't "enabled" if checking in the software section of the installer. Not sure if it is expected.